src/main/java/com/google/cloud/hadoop/io/bigquery/hive/WrappedBigQueryAvroInputFormat.java

```diff
@@ -120,9 +142,9 @@
         /**
          * @param mapreduceInputSplit An InputSplit that also
          *        implements Writable.
-         * @param path A HCFS path of that split. Hive assumes tables are file-based.
+         * @param path A HDFS path of that split. Hive assumes tables are file-based.
```

Comment:
Not typo. HCFS means Hadoop Compatible File System.

Reply:
Got it. Reverted to HCFS